modules/generator/processor/servicegraphs/config.go

```diff
@@ -65,5 +70,9 @@ func (cfg *Config) RegisterFlagsAndApplyDefaults(string, *flag.FlagSet) {
 	}
 	cfg.PeerAttributes = peerAttr
 
+	// Set default database name attributes to support both old and new semantic conventions
+	// Note: "db.system.name" is not yet available as a constant in semconv v1.25.0
```

Comment:
Thanks for the update here.  My suggestion was to update the version we import at the top of this file, so that we can refer to the upstream constant.

Reply:
thanks for the feedback @zalegrala so i checked and updating to a newer semconv version to use DBSystemNameKey so DBSystemNameKey constant is available in v1.34.0 but this version has breaking changes where DBNameKey used elsewhere in the code was replaced with DBNamespaceKey as part of the semantic conventions evolution dbname → dbnamespace shoud i do it?

Reply:
I think the change you are targeting is against 1.30 if I see correctly, so I don't think we need to go to 1.34.0 for the breaking change.  Perhaps a follow up PR could include just the breaking change if we need it.